extensions/notebook/src/extension.ts

```diff
@@ -25,8 +25,12 @@ let counter = 0;
 export let controller: JupyterController;
 
 export function activate(extensionContext: vscode.ExtensionContext) {
-	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.new', (connectionId?: string) => {
-		newNotebook(connectionId);
+	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.new', (context?: azdata.ObjectExplorerContext) => {
```

Comment:
What's the impact here for SSMS???

Reply:
Once we understand the impact (if any) happy to sign-off. Just need to clarify that.

Reply:
the current commandLineService only passes the connection id because a ConnectionProfile instance from the main process wasn't json-serializable to pass to the renderer. You could either create a new version of this command for the commandLineService to invoke, or change the command line service to create a stripped down serializable IConnectionProfile

self._connectionManagementService.connectIfNotConnected(profile, 'connection', true)
						.then(() => {
							self._commandService.executeCommand(commandName, profile.id).then(() => resolve(), error => reject(error));
						}, error => {
							reject(error);
						});

Reply:
or can the command handler dynamically check the type of the parameter?